src/main/java/com/minecolonies/entity/ai/basic/AbstractEntityAISkill.java

```diff
@@ -4,7 +4,9 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)